addons/mail/static/src/utils/messaging_component.js

```diff
@@ -32,6 +34,7 @@ export function registerMessagingComponent(ComponentClass, { propsCompareDepth =
             // fields accessed during useRenderedValues should be observed by
             // useModels as if they were part of the OWL rendering itself.
             useModels();
+            this.root = useRef('root');
```

Comment:
I would define it above our custom hooks in case we need to use it there (and we probably will need it later to save as field).